cpp/ql/src/Documentation/CommentedOutCode.qll

```diff
@@ -20,6 +20,18 @@ private predicate looksLikeCode(string line) {
     exists(string trimmed |
            trimmed = line.regexpReplaceAll("(?i)(^\\s+|&#?[a-z0-9]{1,31};|\\s+$)", "") |
            trimmed.regexpMatch(".*[{};]")
+           and (
+              // If this line looks like code because it ends with a closing
+              // brace that's preceded by something other than whitespace ...
+              trimmed.regexpMatch(".*.\\}")
```

Comment:
Is `.*.` any different to `.*`?

Reply:
Yes: `.*.` is a very silly way of writing `.+`. Do you want me to change that, or can we merge this as it is?

Reply:
Happy to merge it as-is.